### PR TITLE
SASS-3367 added was tax taken off your interest page

### PR DESCRIPTION
**Pull request by kannanramakrishnan**
### Description
Created 'Was tax taken off your interest?' page

Add a link to the relevant story in Jira
[SASS-3367](https://jira.tools.tax.service.gov.uk/browse/SASS-3367)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/views/savings/TaxTakenFromInterestView.scala.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+formWithCsrf: FormWithCSRF,
+errorSummary: ErrorSummary,
+button: Button
+)
+
+@(form: Form[Boolean], taxYear: Int)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{messages(s"savings.tax-taken-from-interest.radio.heading.${if(user.isAgent) "agent" else "individual"}")}
+
+<!--@headingForUse = @{(titleForUse)}-->
```
Comment (satheeshgottemukkula) [inline app/views/savings/TaxTakenFromInterestView.scala.html:36]: please Remove this

File: app/views/savings/TaxTakenFromInterestView.scala.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ * limitations under the License.
+ *@
+
+@import views.html.headerFooterTemplate.Layout
+@import views.html.templates.helpers.ErrorSummary
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.InputRadio
+@import views.html.templates.helpers.Heading
+
+@this(
```
Comment (satheeshgottemukkula) [inline app/views/savings/TaxTakenFromInterestView.scala.html:23]: change Code alignment

File: app/views/savings/TaxTakenFromInterestView.scala.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+@layout(pageTitle = titleForUse, taxYear = Some(taxYear), hasErrors = hasFormErrors, isAgent = user.isAgent) {
+
+@errorSummary(form.errors.distinct)
+
+@heading(headingForUse, Some(messages("savings.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-3")
+
+@formWithCsrf(action = controllers.savings.routes.TaxTakenFromInterestController.submit(taxYear)){
+
+@inputRadio(
```
Comment (satheeshgottemukkula) [inline app/views/savings/TaxTakenFromInterestView.scala.html:48]: Same as above, Please change this code format to match with the standards

**Review by satheeshgottemukkula — CHANGES_REQUESTED** [review]

File: app/views/savings/TaxTakenFromInterestView.scala.html (hunk 1 above)
Comment (kannanramakrishnan) [reply app/views/savings/TaxTakenFromInterestView.scala.html:36]: Removed the commented line

File: app/views/savings/TaxTakenFromInterestView.scala.html (hunk 2 above)
Comment (kannanramakrishnan) [reply app/views/savings/TaxTakenFromInterestView.scala.html:23]: changed alignment

File: app/views/savings/TaxTakenFromInterestView.scala.html (hunk 3 above)
Comment (kannanramakrishnan) [reply app/views/savings/TaxTakenFromInterestView.scala.html:48]: Completed this alignment change as well

**Review by satheeshgottemukkula — APPROVED** [review]

**Pull request merged**